### PR TITLE
KAFKA-7139 Support to exclude the internal topics in kafka-topics.sh …

### PR DESCRIPTION
**Pull request by chia7712**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: core/src/main/scala/kafka/admin/TopicCommand.scala
Comment (andrasbeni) [inline core/src/main/scala/kafka/admin/TopicCommand.scala]: - Does this also work for --describe? If so, it should be stated in here.
- There is a typo in `internal`
- There are excess `the`s. 
- Can you please capitalize the beginning of the second sentence?

Comment (andrasbeni) [inline core/src/main/scala/kafka/admin/TopicCommand.scala]: I believe this flag does not make sense for commands other than --describe and --list (because they require --topic anyway ). Could you please also add a check to checkArgs() ?

**Review by andrasbeni — CHANGES_REQUESTED** [review]
@chia7712 This pull request modifies command line arguments, which are considered public interface. For such changes a [KIP](https://cwiki.apache.org/confluence/display/KAFKA/Kafka+Improvement+Proposals) is required.

File: core/src/main/scala/kafka/admin/TopicCommand.scala
Comment (chia7712) [reply core/src/main/scala/kafka/admin/TopicCommand.scala]: Thanks for the reviews! I will address your comment after completing the KIP flow.

Comment (chia7712) [reply core/src/main/scala/kafka/admin/TopicCommand.scala]: copy that.

**Comment by chia7712** [comment]
> This pull request modifies command line arguments, which are considered public interface. For such changes a KIP is required.

(This Q is unrelated to this PR) How to know what is public Interface? For example, Does [TopicAdmin](https://github.com/apache/kafka/blob/trunk/connect/runtime/src/main/java/org/apache/kafka/connect/util/TopicAdmin.java) has public interface?

**Comment by andrasbeni** [comment]
In general, you can refer to [What is considered a "major change" that needs a KIP?](https://cwiki.apache.org/confluence/display/KAFKA/Kafka+Improvement+Proposals#KafkaImprovementProposals-Whatisconsidereda%22majorchange%22thatneedsaKIP?), but as I see, it does not list any Connect-related packages. 
I am not particularly familiar with Kafka Connect, so I think someone who contributes there more (e.g. @ewencp ) is much more suited to answer this question.

**Comment by chia7712** [comment]
@andrasbeni Got it. Thanks for the explanation.

**Comment by chia7712** [comment]
@andrasbeni all comments are addressed. Please take a look. thanks!

**Comment by andrasbeni** [comment]
LGTM

**Comment by chia7712** [comment]
@andrasbeni Thanks for the reviews.

**Comment by chia7712** [comment]
@andrasbeni Could we merge this PR?

**Comment by andrasbeni** [comment]
@chia7712 I can not, because I'm not a committer.
Hopefully @hachikuji or @ijuma have the bandwidth to take a look and merge

**Comment by chia7712** [comment]
@andrasbeni Thanks for the response :)

File: core/src/test/scala/unit/kafka/admin/TopicCommandTest.scala
Comment (hachikuji) [inline core/src/test/scala/unit/kafka/admin/TopicCommandTest.scala]: Should this be `--list`?

**Review by hachikuji — COMMENTED** [review]
Thanks for the patch. Noticed one typo, but otherwise looks great.

File: core/src/test/scala/unit/kafka/admin/TopicCommandTest.scala
Comment (chia7712) [reply core/src/test/scala/unit/kafka/admin/TopicCommandTest.scala]: thanks for catching!!

**Review by hachikuji — APPROVED** [review]
Thanks, LGTM. I will merge after the build finishes.

**Pull request merged**
